Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/AdapterBuilder.cpp

```diff
@@ -24,7 +24,7 @@ namespace AZ::DocumentPropertyEditor
     void AdapterBuilder::OnEditorChanged(AZStd::function<void(const Dom::Path&, const Dom::Value&)> onChangedCallback)
     {
         // The value path is the first child of the PropertyEditor node
-        Dom::Path changedPath = GetCurrentPath() / 0;
+        Dom::Path changedPath = GetCurrentPath() / Nodes::Label::Value.GetName();
```

Comment:
@NicholasVanSickle Why is 0 allowed to work here?

Reply:
`Dom::Path` is comprised with `size_t` indices or `AZ::Name` entries (to represent indexing by child position or attribute respectively) so `/ 0` was a perfectly valid, if somewhat surprising, usage.